README.md

```diff
@@ -1,105 +1,137 @@
+| Level 1        				| 100          		| 190        		| 240			|
+| Level 2        				| 100          		| 220	      		| 280			|
+| Level 3        				| 100          		| 250        		| 320			|
+| Level 4        				| 100          		| 280        		| 360			|
 
-- Level 0 = We have less than 1 month worth of employees base salary in the bank.
-- Level 1 = We have 1 month worth of employees base salary in the bank.
-- Level 2 = We have 2 months worth of employees base salary in the bank.
-- Level 3 = We have 3 months worth of employees base salary in the bank.
-- Level 4 = We have 4 months worth of employees base salary in the bank.
+### Forcasting example 
 
+Given we are planning for febuary of a non leap year
+And we are at Buffer Level 2
+And a Member has booked in:
 
-#### Charge Out Rate
-How much a member costs per hour to the client.
+ - 4 Personal days
+ - 6 Internal days
+ - 10 Contracting days
+ 
+And we can estimat:
 
-#### Billable Hours
-The number of hours a member was billable in a month.
+ - 6 billable hours contracting day
+ - a charge out rate of $140
 
----
+When we run a cashflow forecast
+Then we can calculate the member will recieve:
 
-### Buffer events (WIP)
-Moving up or down the scale is seen as an event which will have side effects. Side effects may be defined in other agreements and/or commitments. Buffer transitions should be recorded in a commitment detailing the side-effects in a migration style format. Side-effects should have links to any pertaining information. eg:
 
-```markdown
-## Buffer 1
+                     4 * 100
+                     4 * 220
+                    10 * 280
+              60 * 0.1 * 100
+            ----------------
+                 Total: 5120
 
-### Up
-- [side effects that occur when buffer moves to a level 2 buffer](www.link.to.it).
+### End of Month
+At the end of a month we produce the invoices for our members. This follows the same process as forecasting except that we substitute in the actual billable hours worked and charge out rate.
 
-### Down
-- [side effects that occur when buffer 1is moving down to no buffer](www.link.to.it).
-```
+### !Important (very experimental)
+We do not adjust peoples pay down because of changes to their schedule. eg if a person gets sick and cant work we don't demote there days to personal. We will adjust peoples pay up if they have had to do extra days they didn't intend to.
+
+It is the job of the Coordinators to keep an eye on this overtime and checkin with the group and members about how things are going. A transparent monthly report will be shared to all members.
 
 ### Buffer as control lever (What's the point of all this?)
 By coupling several factors to buffer events we can create a self regulating system. The intent is that as the buffer increases so to does the money flowing away from the pod. These are the scaling money outflows and do not include fixed costs such as space hire and saas tools.
 
+## Contribution to a commons
```

Comment:
It is not committing us to anything rather there as an indication. We have talked about making an Enspiral contribution - but haven't for what ever reason.